src/Datasource/composer.json

```diff
@@ -26,7 +26,7 @@
     "require": {
         "php": ">=7.2.0",
         "cakephp/core": "^4.0",
-        "psr/log": "^1.1",
+        "psr/log": "^2 || ^1.1",
```

Comment:
Can u please fix the order?

Reply:
So ` "psr/log": "^1.1 || ^2",` instead of ` "psr/log": "^2 || ^1.1",`?